website/snippets/core-versions-table.md

```diff
@@ -5,9 +5,9 @@
 | [**v1.5**](/guides/migration/versions/upgrading-to-v1.5)   | Apr 27, 2023    | Active        | Apr 27, 2024            | Apr 2024        |
 | [**v1.4**](/guides/migration/versions/upgrading-to-v1.4)   | Jan 25, 2023    | Critical      | Jan 25, 2024            | Jan 2024        |
 | [**v1.3**](/guides/migration/versions/upgrading-to-v1.3)   | Oct 12, 2022    | Critical      | Oct 12, 2023            | Oct 2023        |
-| [**v1.2**](/guides/migration/versions/upgrading-to-v1.2)   | Jul 26, 2022    | Critical      | Jul 26, 2023            | Jul 2023        |
-| [**v1.1**](/guides/migration/versions/upgrading-to-v1.1) ⚠️ | Apr 28, 2022    | End of Life ⚠️ | Apr 28, 2023            | Apr 2023        |
-| [**v1.0**](/guides/migration/versions/upgrading-to-v1.0) ⚠️ | Dec 3, 2021     | End of Life ⚠️ | Dec 3, 2022 ⚠️           | Pending deprecation|
+| [**v1.2**](/guides/migration/versions/upgrading-to-v1.2)   | Jul 26, 2022    | Critical      | Jul 26, 2023            | Supported until further notice |
```

Comment:
clarifying question: v1.2 is at `Critical` support level not EOL. will "Supported until further notice" be confusing?

Reply:
log: resolved in slack